documentation/domains/Cluster.json

```diff
@@ -509,7 +529,7 @@
           "type": "boolean"
         },
         "serverPod": {
-          "description": "Customization affecting the generation of Pods for WebLogic Server instances.",
+          "description": "Customization affecting the generation of Pods for WebLogic Server instances or Introspector Job Pod.",
```

Comment:
"Introspector Job Pod' doesn't look like it belongs in the cluster scope.

Reply:
There's a general issue with Cluster.json where all domain level settings are included in Cluster.json. This needs to be investigated separately.

Reply:
Fixed in [a6b2383](https://github.com/oracle/weblogic-kubernetes-operator/pull/3914/commits/a6b23837e674e333f027e573edba8c9dedd57ca5).